cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -558,7 +558,9 @@ are case sensitive and MUST all be handled by `/serviceValidate`.
     primary credentials. It will fail if the ticket was issued from a single
     sign-on session.
 
-
+-   `format` [OPTIONAL] - if this parameter is set, ticket validation response
+    MUST be produced based on the parameter value. Supported values are `XML`
+    and `JSON`. If this parameter is not set, the default `XML` format will be used. 
```

Comment:
We must add that if the format is not supported (by the CAS server or the service): a 400 bad request error must be returned. So that CAS clients can start supporting that.